yarn-project/end-to-end/src/integration_l1_publisher.test.ts

```diff
@@ -266,7 +267,14 @@ describe('L1Publisher integration', () => {
         await makeBloatedProcessedTx(128 * i + 96),
         await makeBloatedProcessedTx(128 * i + 128),
       ];
-      const [block] = await builder.buildL2Block(1 + i, txs, l1ToL2Messages);
+      // @todo @LHerskind fix time.
```

Comment:
this stale?

Reply:
No, its part of a separate pr that is to fix #830, to be fixed in #948